rcjaRegistration/events/admin.py

```diff
@@ -152,7 +185,7 @@ def get_exclude(self, request, obj=None):
     def fkObjectFilterFields(cls, request, obj):
         return {
             'division': {
-                'queryset': Division.objects.filter(Q(state=obj.state) | Q(state=None)) if obj is not None else Division.objects.none(), # Inline not displayed on create so user will never see fallback to None
+                'queryset': Division.objects.filter((Q(state=obj.state) | Q(state=None))& Q(active=True)) if obj is not None else Division.objects.none(), # Inline not displayed on create so user will never see fallback to None
```

Comment:
Where the field we are filtering on is static, can use limit_choices_to in the model defiition.